doc/examples/segment_quickbundles.py

```diff
@@ -30,7 +30,7 @@
 
 streams, hdr = tv.read(fname)
```

Comment:
We should be using `dipy.io.streamline.load_trk` instead of `tv.read`. The best part is that `load_trk` returns a `Streamlines` object and the header. That means, there is no need for line 33 at all :-).

Reply:
@skoudoro mentioned that if you use dipy.io.streamline.load_trk to read the dataset of 'fornix', will cause some problem. This may need some further effort beyond my knowledge right now. So I just leave this problem there.

Reply:
@skoudoro what's the problem exactly?

Reply:
`fornix.trk` has an old header that misses some information @MarcCote. I do not remember which information exactly is missing but it crashes when you try to load it with `dipy.io.streamline.load_trk`. (more details, tomorrow :-) )

 We just need to update this data. I plan to do it on a new PR.

Reply:
Ok, I see. It seems the header of the fornix file was zeroed out.